desktop/hardened-desktop/settings-management/configure.md

```diff
@@ -112,6 +113,7 @@ The following `admin-settings.json` code and table provides an example of the re
 | `configurationFileVersion`        |   |Specifies the version of the configuration file format.    |
 | `exposeDockerAPIOnTCP2375` | <span class="badge badge-info">Windows only</span>| Exposes the Docker API on a specified port. If `value` is set to true, the Docker API is exposed on port 2375. Note: This is unauthenticated and should only be enabled if protected by suitable firewall rules.| 
 | `proxy` |   |If `mode` is set to `system` instead of `manual`, Docker Desktop gets the proxy values from the system and ignores and values set for `http`, `https` and `exclude`. Change `mode` to `manual` to manually configure proxy servers. If the proxy port is custom, specify it in the `http` or `https` property, for example `"https": "http://myotherproxy.com:4321"`. The `exclude` property specifies a comma-separated list of hosts and domains to bypass the proxy. |
+&nbsp; &nbsp; &nbsp; &nbsp;`windowsDockerdPort`  | <span class="badge badge-info">Windows only</span> | Expose Docker Desktop's internal proxy locally on this port for the Windows Docker daemon to connect to. If it is set to 0, a random free port will be chosen. If the value is > 0, use that exact value for the port. The default value is -1 and it means that the option is disabled. Note: This is available for Windows containers only. |
```

Comment:
```suggestion
&nbsp; &nbsp; &nbsp; &nbsp;`windowsDockerdPort`  | <span class="badge badge-info">Windows only</span> | Exposes Docker Desktop's internal proxy locally on this port for the Windows Docker daemon to connect to. If it is set to 0, a random free port is chosen. If the value is greater than 0, use that exact value for the port. The default value is -1 which disables the option. Note: This is available for Windows containers only. |
```

Reply:
I'd actually remove the note. We have the label which signals it's only for Windows and arguably stands out more and is consistent with how we've done it elsewhere in the table.

Reply:
What about `When using Windows containers, exposes ...`?

Reply:
Oh! Please ignore my earlier comment. I hadn't had my second coffee of the day and didn't register the word 'containers'. So sorry. Just those small changes to make the sentence more active and it'll be good to go